apps/stirling-pdf/0.36.0/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   stirling-pdf:
-    image: frooodle/s-pdf:0.36.0
+    image: frooodle/s-pdf:0.36.1
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
The code looks to be updating the version of an image used in a Docker Compose configuration. The key difference is that it's changing from `frooodle/s-pdf:0.36.0` to `frooodle/s-pdf:0.36.1`. This suggests that there might have been some changes or updates released with this new version of the Docker image. If these updates address bugs, performance improvements, or additional features, then using the new version could benefit your application.

Here's what you should consider:

1. **Compatibility**: Check if this updated version maintains compatibility with your existing setup.
2. **Testing**: Always test any update in a staging environment before deploying it to production to ensure no unintended side effects.
3. **Monitoring**: Implement monitoring to track the performance and behavior of the service after the update.

If you decide to proceed with the update, make sure to validate all other configurations and variables related to this service (like `container_name`) to ensure they still work correctly with the updated image.
**No further actions needed regarding specific code snippets provided, focus only on the logical reasoning based on the provided text details.**